Project Website/assets/js/main.js

```diff
@@ -938,4 +938,4 @@ function randomImage() {
   $('.myimage').append("<img src='" + path + imgs[i] + "'>").hide().fadeIn(2000);
 }
 
-randomImage();
```

Comment:
This line of code has met some changes. Please consider reverting to the original.

Reply:
The same line has been removed and added as shown in the screenshot.
If still, you think that I should revert it to the original, please guide me, sir.

![Fix #659 by priyaastha · Pull Request #718 · Tejas1510_Hacking-Scripts - Google Chrome 25-06-2021 21_33_35](https://user-images.githubusercontent.com/55706563/123453329-0a3acd00-d5fd-11eb-82b6-87b409ddad55.png)